org.osgi.test.junit5.cm/src/main/java/org/osgi/test/junit5/cm/package-info.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) OSGi Alliance (2019). All Rights Reserved.
```

Comment:
Minor nit: there are a bunch of cut-and-paste errors here - some are (2019), others are (2019, 2020). All of them in this PR should be just (2020).

Reply:
ok